Tools/autotest/arduplane.py

```diff
@@ -4515,7 +4515,7 @@ def AerobaticsScripting(self):
         self.customise_SITL_commandline(
             [],
             model=model,
-            defaults_filepath="",
+            defaults_filepath="Tools/autotest/models/plane-3d.parm",
```

Comment:
What's this doing here?

Reply:
it is needed as I changed the AP_Filsystem::stat() call to behave the same as ChibiOS, so that lua scripts behave the same on ChibiOS and SITL. I noticed they didn't match when developing the web server
That autotest was using a param file of "" which maps to a directory, which is not legal to open any more, so needed a fix